app/assets/javascripts/admin/controllers/admin-site-settings.js.es6

```diff
@@ -2,6 +2,8 @@ import debounce from "discourse/lib/debounce";
 
 export default Ember.Controller.extend({
   filter: null,
+  allSiteSettings: Ember.computed.alias("model"),
+  visible: null,
```

Comment:
To me `visible` is a boolean, not a list of visible settings. Maybe call it `visibleSiteSettings`?

Reply:
👍 done